src/test/HostActivationTests/GivenThatICareAboutMultilevelSharedFxLookup.cs

```diff
@@ -144,125 +155,164 @@ public void SharedFxLookup_Must_Verify_Folders_in_the_Correct_Order()
             dotnet.Exec(appDll)
                 .WorkingDirectory(_currentWorkingDir)
                 .EnvironmentVariable("COREHOST_TRACE", "1")
-                .WithUserProfile(_userDir)
+                .WithGlobalLocation(_globalDir)
                 .CaptureStdOut()
                 .CaptureStdErr()
                 .Execute()
                 .Should()
                 .Pass()
                 .And
-                .HaveStdErrContaining(Path.Combine(_userSelectedMessage, "9999.0.0-dummy1"));
+                .HaveStdErrContaining(Path.Combine(_exeSelectedMessage, "9999.0.0-dummy0"));
 
-            // Set desired version = 9999.0.0
             SetRuntimeConfigJson(runtimeConfig, "9999.0.0");
-
             // Version: 9999.0.0
-            // CWD: 9999.0.0-dummy1
-            // User: 9999.0.2, 9999.0.0-dummy2
             // Exe: 9999.0.0, 9999.0.0-dummy0
-            // Expected: 9999.0.2 from user dir
+            // global: 9999.0.2, 9999.0.0-dummy1, 9999.0.0-dummy2
+            // Expected: 9999.0.2 from global
             dotnet.Exec(appDll)
                 .WorkingDirectory(_currentWorkingDir)
                 .EnvironmentVariable("COREHOST_TRACE", "1")
-                .WithUserProfile(_userDir)
+                .WithGlobalLocation(_globalDir)
                 .CaptureStdOut()
                 .CaptureStdErr()
                 .Execute()
                 .Should()
                 .Pass()
                 .And
-                .HaveStdErrContaining(Path.Combine(_userSelectedMessage, "9999.0.2"));
+                .HaveStdErrContaining(Path.Combine(_globalSelectedMessage, "9999.0.2"));
 
-            // Remove dummy folders from user dir
-            DeleteAvailableSharedFxVersions(_userSharedFxBaseDir, "9999.0.2", "9999.0.0-dummy2");
+           AddAvailableSharedFxVersions(_exeSharedFxBaseDir, "9999.0.3");
+
+            // Version: 9999.0.0
+            // Exe: 9999.0.0, 9999.0.0-dummy0, 9999.0.3
+            // global: 9999.0.2, 9999.0.0-dummy1, 9999.0.0-dummy2
+            // Expected: 9999.0.0-dummy1 from global
+            dotnet.Exec(appDll)
+                .WorkingDirectory(_currentWorkingDir)
+                .EnvironmentVariable("COREHOST_TRACE", "1")
+                .WithGlobalLocation(_globalDir)
+                .CaptureStdOut()
+                .CaptureStdErr()
+                .Execute()
+                .Should()
+                .Pass()
+                .And
+                .HaveStdErrContaining(Path.Combine(_exeSelectedMessage, "9999.0.3"));
+
+            DeleteAvailableSharedFxVersions(_exeSharedFxBaseDir, "9999.0.0", "9999.0.0-dummy0","9999.0.3");
+            DeleteAvailableSharedFxVersions(_globalSharedFxBaseDir, "9999.0.2", "9999.0.0-dummy1", "9999.0.0-dummy2");
         }
 
         [Fact]
         public void SharedFxLookup_Must_Not_Roll_Forward_If_Framework_Version_Is_Specified_Through_Argument()
         {
+            if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Windows)
```

Comment:
With the removal of userprofile the only other probe location is the global location on windows, and it is not easy to mock it unless we add knobs into the host itself.

I believe once the tracking item on containarizing the tests https://github.com/dotnet/core-setup/issues/1553 is done, then we can make the infra changes necessary for enabling these tests on windows